### PR TITLE
Create 2 page

### PR DESCRIPTION
**Pull request by jxah**

### CODE REVIEW

File: help/2 page [hunk 1]
```diff
@@ -0,0 +1 @@
+teach me papa
```
Comment (maparce) [inline help/2 page:1]: Happy to!

Comment (jxah) [inline help/2 page:1]:
```suggestion
teach me papa
```

Comment (jxah) [reply help/2 page:1]: - [ ] how to put 2 pages in 1
- [ ] how to make paths

**Review by jxah — COMMENTED** [review]
@maparce **approve** [this](#1 ) _now_ `please`

**Review by maparce — APPROVED** [review]
Chevere!

File: help/2 page (hunk 1 above)
Comment (maparce) [reply help/2 page:1]: Do you mean two HTML pages?

Comment (jxah) [reply help/2 page:1]: yes

Comment (jxah) [reply help/2 page:1]: both

Comment (maparce) [reply help/2 page:1]: Do you want to make paths in SVG or in CANVAS?  I am guessing is in Canvas since it is for Solid Happiness.

**Pull request merged**